apache/superset
# fix: Allow setting ingress hosts directly through yaml

SUMMARY
Fixes https://github.com/apache/superset/issues/23445

This PR adds the ability to configure ingress hosts as a raw block of YAML

BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

TESTING INSTRUCTIONS
Set `ingress.extraHostsRaw` to a valid yaml configuration in the values file

ex:
```
ingress:
  extraHostsRaw:
    - host: superset.myhost.com
      http:
        paths:
          - path: /
            pathType: Prefix
            backend:
              service:
                name: ssl-redirect
                port:
                  name: use-annotation
          - path: /*
            pathType: ImplementationSpecific
            backend:
              service:
                name: superset
                port:
                  name: http
```
Deploy using Helm
Observe the ingress is created with the specified path configuration.


ADDITIONAL INFORMATION

- [ ]  Includes DB Migration (follow approval process in https://github.com/apache/superset/issues/13351)
